plugins/tekton/README.md

```diff
@@ -34,6 +36,14 @@ The Tekton plugin enables you to visualize the `PipelineRun` resources available
     metadata:
       name: backstage-read-only
     rules:
+      - apiGroups:
+          - ""
+        resources:
+          - pods/log
```

Comment:
is this needed, I don't think there is a resource `pod/log`.

cc @debsmita1

Reply:
I wasn't able to get the logs to load without adding `pods/log` into the `ClusterRole`

Reply:
ah yeah my bad, this would be needed. 👍